Mathlib/RepresentationTheory/Rep.lean

```diff
@@ -424,7 +424,6 @@ def MonoidalClosed.linearHomEquiv : (A ⊗ B ⟶ C) ≃ₗ[k] B ⟶ A ⟶[Rep k
   { (ihom.adjunction A).homEquiv _ _ with
     map_add' := fun _ _ => rfl
     map_smul' := fun _ _ => rfl }
-  set_option linter.uppercaseLean3 false in
```

Comment:
FWIW, #14914 also includes this: you certainly need not search for all those comments. (Probably, this won't cause a conflict either - well, at least I hope so.)